appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/WebServiceInvocationHandler.java

```diff
@@ -99,27 +89,44 @@ public WebServiceInvocationHandler(Class ejbClass,
         invocationInfoMap_ = invocationInfoMap;
     }
 
-    public Object invoke(Object proxy, Method method, Object[] args) 
-        throws Throwable {
+
+    @Override
+    public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
         try {
-        container.onEnteringContainer();
-        // NOTE : be careful with "args" parameter.  It is null
-        //        if method signature has 0 arguments.
-
-        Class methodClass = method.getDeclaringClass();
-        if( methodClass == java.lang.Object.class )  {
-            return InvocationHandlerUtil.
-                invokeJavaObjectMethod(this, method, args);    
+            container.onEnteringContainer();
+            // NOTE : be careful with "args" parameter. It is null
+            // if method signature has 0 arguments.
+            final Class<?> methodClass = method.getDeclaringClass();
+            if (methodClass == java.lang.Object.class) {
+                return InvocationHandlerUtil.invokeJavaObjectMethod(this, method, args);
+            }
+
+            // Invocation was created earlier in the web service dispatching
```

Comment:
Question to github: where is the - section?

Reply:
Aha, response: I extracted part of the method to a standalone private method. That confused even author as a reviewer :dancing_men: